core/simulations/config.ts

```diff
@@ -1,5 +1,11 @@
 import debtAuctionSimulation from './configs/createDebtAuctionSimulation';
-import surplusAuctionSimulation from './configs/surplusAuctionSimulation';
+import surplusAuctionBlockchain from './configs/surplusAuctionSimulation';
 import wstethAuctionSimulation from './configs/wstethAuctionSimulation';
+import surplusAuctionSimulation from './configs/createSurplusAcutionSimulation';
 
-export const SIMULATIONS = [debtAuctionSimulation, surplusAuctionSimulation, wstethAuctionSimulation];
+export const SIMULATIONS = [
```

Comment:
One thing that I missed last time: let's create a type that those configs should adhere to and add it here